client/base.jsx

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * Module dependencies.
+ */
+
+var React = require('react');
+
+module.exports = React.createClass({
+	render: function() {
+
+		var googleMapsScript =
+		    'function initialize() { dispatcher.dispatch(\'GOOGLE_MAPS_READY\'); } ' +
+			'google.maps.event.addDomListener(window, \'load\', initialize);';
+
+		var googleMapsBootstrap = '';
+
+		if (this.props.useGoogleMaps) {
+			googleMapsBootstrap = [
```

Comment:
What's the difference between this and the parens? Is it because there are multiple root elements?